app/main/templates/home.html

```diff
@@ -94,5 +119,24 @@ <h2 class="section-title">How do I submit an application?</h2>
       </div>
       <label >****NOTE: Incomplete Applications will NOT be considered and will be returned****</label>
     </div>
+    <br>
+    <center>
+    <div id="carousel">
```

Comment:
Dont use <br> or <center>; do the markup purely in CSS for your #carousel

Use top-padding and "margin: 0 auto;" to recreate the same thing.